src/OpenTelemetry.Exporter.NewRelic/NewRelicTraceExporter.cs

```diff
@@ -87,9 +87,12 @@ internal NewRelicTraceExporter(SpanDataSender spanDataSender, TelemetryConfigura
         /// <inheritdoc />
         public override ExportResult Export(in Batch<Activity> batch)
         {
+            // Prevent exporter's HTTP operations from being instrumented.
+            using var scope = SuppressInstrumentationScope.Begin();
```

Comment:
Could we consider using the braces here?  I was unaware that the C# 8.0 feature allows this.  May pose difficulty in understanding.

Reply:
I'm seeing it become more commonplace. I personally think it's good for people to become accustomed to. Reduces nesting, so that's a small benefit.